src/compiler/transformers/module/module.ts

```diff
@@ -820,6 +831,14 @@ namespace ts {
             let statements: Statement[];
             let variables: VariableDeclaration[];
             let expressions: Expression[];
+
+            const parseTreeNode = getParseTreeNode(node);

```

Comment:
@rbuckton After some thought, I think we should emit one here. Though in the following case, I don't think we should. By checking `!currentModuleIndo.exportEqual`, we will emit the marker still. 
```ts
import * as Foo from "Foo"
```